src/style.css

```diff
@@ -1,3 +1,91 @@
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  --main-color: #00acdc;
+}
+/*In general we should use "!important" very carefully.
+But in case of hidden element it useful*/
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  background-color: #fff;
+  justify-content: flex-end;
+}
+
+.logo {
+  display: flex;
+  position: absolute;
+  left: 50px;
+  align-self: center;
+}
+
+.nav {
+  position: relative;
+  right: 50px;
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: center;
+  padding: 0;
+  margin: 0;
+}
+
+.list_item {
+  box-sizing: border-box;
+  display: inline-block;
+  list-style: none;
+  height: 60px;
+  line-height: 60px;
```

Comment:
you can center text with flexbox or vertical padding, no need to set `line-height` here

Reply:
not fixed